packages/babylon/rollup.config.js

```diff
@@ -1,34 +0,0 @@
-import babel from "rollup-plugin-babel";
-import nodeResolve from "rollup-plugin-node-resolve";
-
-export default {
-  input: "src/index.js",
-  output: {
-    file: "lib/index.js",
-    format: "cjs",
```

Comment:
there was no deps though so I don't think it was any different for the format?

Reply:
It still has exports.

Reply:
ah yeah I just meant no requires